scrapers/dlsite.yml

```diff
@@ -0,0 +1,100 @@
+name: DLsite
+sceneByFragment:
+  action: scrapeXPath
+  queryURL: https://www.dlsite.com/maniax/work/=/product_id/{title}.html
+  #queryURL: https://www.dlsite.com/maniax/work/=/product_id/{title}.html/?locale=en_US
+  scraper: sceneScraper
```

Comment:
This expects that the title is exactly the same as the sites title?
Won't that create too much trouble especially for users that dont know that a specific title is needed?
Shouldnt we remove all the the `sceneByFragment` section since there is a `sceneByName/sceneByQueryFragment` that works better?

Reply:
The expectation was that someone would have the title as RJxxxx. I was mainly thinking of identify users.

Reply:
I am not familiar with the site. Do the downloaded files have the id/RJxxx as their filename or is the user supposed to name the files like that? It should be ok, just add an extra comment of the expected title/filename for the scenebyfragment  to work

Reply:
It would be reasonable for a user to have a file that was just RJxxxx as the filename as that's how works from DLsite are generally referred to (similar to JAV codes). Happy to leave a comment clarifying that it'll need the RJ code as the title.